### PR TITLE
Fix heroku review app deployment

### PR DESCRIPTION
**Pull request by vkrmis**
**Story card:** [ch6279](https://app.shortcut.com/simpledotorg/story/6279/switching-to-structure-sql-broke-our-heroku-review-app-deployments)

Switching from `schema.db` to `structure.sql` is breaking heroku review app deployment.

### CODE REVIEW

File: db/structure.sql [hunk 1]
```diff
@@ -9,48 +9,20 @@ SET xmloption = content;
 SET client_min_messages = warning;
 SET row_security = off;
 
---
--- Name: plpgsql; Type: EXTENSION; Schema: -; Owner: -
---
-
-CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
```
Comment (prabhanshuguptagit) [inline db/structure.sql:16]: This is extra?

**Review by prabhanshuguptagit — APPROVED** [review]

File: db/structure.sql (hunk 1 above)
Comment (vkrmis) [reply db/structure.sql:16]: I think its disallowed, I'll double check

Comment (vkrmis) [reply db/structure.sql:16]: It actually works fine, re-including it

File: Rakefile [hunk 2]
```diff
@@ -7,3 +7,8 @@ Rails.application.load_tasks
 def is_running_migration?
   Rake.application.top_level_tasks.include?("db:migrate")
 end
+
+Rake::Task["db:schema:load"].clear
+task "db:schema:load" do
+  Rake::Task["db:structure:load"].invoke
```
Comment (rsanheim) [inline Rakefile:13]: Why do we need to redefine this task if we manually call the structure load in `heroku-setup` ?

**Review by rsanheim — COMMENTED** [review]
thx for the fix - I guess we have to find a way to keep this in sync after the schema gets dumped ?

**Pull request merged**